c_api/docs/taichi/taichi_core.h.md

```diff
@@ -0,0 +1,410 @@
+---
+sidebar_position: 1
+---
+
+# Core Functionality
+
+Taichi Core exposes all necessary interfaces to offload AOT modules to Taichi. Here lists the features universally available disregards to any specific backend. These APIs are still in active development so is subject to change.
+
+## Availability
+
+Taichi C-API has bridged the following backends:
```

Comment:
The word "bridge" does not sound very clear to me. How about "is supported on the following backends"? Set me straight if I were wrong.